environment_doc.yml

```diff
@@ -7,10 +7,10 @@ dependencies:
   - pip==20.2.4
   - wheel==0.35.1
   - sphinx==3.2.1
-  - dask-sphinx-theme==1.3.5
   - dask==2.8.1
   - numpy==1.15.4
   - pims==0.4.1
   - slicerator==0.9.8
   - pip:
-    - slicerator==0.9.8
+      - slicerator==0.9.8
+      - dask-sphinx-theme>=2
```

Comment:
Would it make sense to still use the Conda package?

Reply:
This is the only project I've seen that uses the conda package. Everywhere else seems to use pip. So this change was partly for consistency, and partly because conda doesn't seem to have v2 yet for some reason.